pkg/util/provider/machinecontroller/machine_safety_util.go

```diff
@@ -0,0 +1,29 @@
+package controller
+
+import (
+	"github.com/gardener/machine-controller-manager/pkg/util/provider/machineutils"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/klog"
+)
+
+func (c *controller) updateNodeWithAnnotation(node *v1.Node, annotations map[string]string) (machineutils.RetryPeriod, error) {
```

Comment:
No need for this func to return the retry period. The caller should decide the retry period (which is a constant here anyway). Does it matter if it is medium or long retry?

Reply:
@amshuman-kr MediumRetry is 2 min, LongRetry is 10 min. I am returning MediumRetry when I'm unable to patch the node object on API Server . A MediumRetry looked more time effective in this case , so I have used it. 
If it doesn't make much difference , then I can remove it.